service/history/queue/processor_base.go

```diff
@@ -374,7 +374,7 @@ func (p *processorBase) resetProcessingQueueStates() (*ActionResult, error) {
 				defaultProcessingQueueLevel,
 				minAckLevel,
 				maxReadLevel,
-				NewDomainFilter(nil, false),
+				NewDomainFilter(nil, true),
```

Comment:
What is this change for?

Reply:
It's a bug fix. The boolean here means whether this domain filter is a reverse match or not. When there's only one queue, the reverse match should always be true (match all domains). Otherwise all tasks will be skipped.